src/Splat.NLog/Splat.NLog.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+  <PropertyGroup>
+    <TargetFrameworks>net461;uap10.0.16299;MonoAndroid80;Xamarin.iOS10;netstandard2.0</TargetFrameworks>
```

Comment:
Only other suggestion I have is adding the platforms from the main Splat project. Not sure if its strictly necessary though since in theory I think .netstandard2.0/uap/net461 will be likely the only ones needed.